velox/common/memory/MappedMemory.cpp

```diff
@@ -244,6 +284,17 @@ int64_t MappedMemoryImpl::free(Allocation& allocation) {
   allocation.clear();
   return numFreed * kPageSize;
 }
+void MappedMemoryImpl::freeContiguous(ContiguousAllocation& allocation) {
+  if (allocation.data()) {
+    if (munmap(allocation.data(), allocation.size()) < 0) {
+      LOG(ERROR) << "munmap returned " << errno << "for " << allocation.data()
+                 << ", " << allocation.size();
+    }
+    numMapped_.fetch_sub(allocation.numPages());
```

Comment:
I see this call to subtract from numMapped_, but I can't see the call to add to it. Can you give me some pointers?